docs/csharp/language-reference/keywords/out-parameter-modifier.md

```diff
@@ -43,14 +43,12 @@ You can't use the `in`, `ref`, and `out` keywords for the following kinds of met
   
 -   Iterator methods, which include a [yield return](../../../csharp/language-reference/keywords/yield.md) or `yield break` statement.  
 
-## Declaring `out` arguments   
+## Declaring `out` parameters   
 
- Declaring a method with `out` arguments is useful when you want a method to return multiple values. The following example uses `out` to return three variables with a single method call. Note that the third argument is assigned to null. This enables methods to return values optionally.  
+Declaring a method with `out` arguments is the classic way to return multiple values. Beginning with C# 7.0 [tuples](../../tuples.md) are preferred. The following example uses `out` to return three variables with a single method call. Note that the third argument is assigned to null. This enables methods to return values optionally.  
   
 [!code-csharp-interactive[cs-out-keyword](../../../../samples/snippets/csharp/language-reference/keywords/in-ref-out-modifier/OutParameterModifier.cs#3)]  
 
- The [Try pattern](/visualstudio/code-quality/ca1021-avoid-out-parameters#try-pattern-methods) involves returning a `bool` to indicate whether an operation succeeded or failed, and returning the value produced by the operation in an `out` argument. A number of parsing methods, such as the [DateTime.TryParse](xref:System.DateTime.TryParse(System.String,System.DateTime@)) method, use this pattern.
```

Comment:
I'm glad you're removing this -- this is a really bizarre recommendation (don't use in or out because they're too complex for most developers to understand). More than a bit condescending.